roles/hello-world-apb/tasks/verify_deprovision.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: "verify deployment config removed"
+  shell: "oc get deploymentconfig -n {{ namespace }} {{ app_name }}"
```

Comment:
I think chouseknecht has added a lookup module for this, probably a good place to showcase it: https://docs.ansible.com/ansible/devel/plugins/lookup/openshift.html